src/test/resources/PollerHealthCheckConfigTest/minimal-config.yml

```diff
@@ -1,3 +1,4 @@
 ---
 dummyValue: "Minimal Config"
 healthCheckConfig:
+  missingPollMultiplier: 12
```

Comment:
Might as well add the newline...can we configure IntelliJ to automatically make sure there is one on commit???